integration-tests/test_client_options.py

```diff
@@ -87,3 +88,21 @@ def test_support(insights_client):
     assert "Running command:" in support_result.stdout
     assert "Process output:" in support_result.stdout
     assert "Support information collected in" in support_result.stdout
+
+
+def test_client_validate_no_network_call(insights_client):
+    """Verify that '--validate' option will not try to connect to any network service.
+
+    Ref : https://bugzilla.redhat.com/show_bug.cgi?id=2009864
+    """
+
+    # '/etc/insights-client/tags.yaml' file exists with any content
+    tags_filename = "/etc/insights-client/tags.yaml"
+    subprocess.run(["touch", tags_filename])
```

Comment:
note that this file must be removed at the end of the test; what you can do is wrap the code of the test in a `try ... finally:` block, and in `finally` remove the file: this way, errors are propagated, and the file is removed in any case

Reply:
done